src/localization/online_localizer/online_localizer.cpp

```diff
@@ -125,110 +125,76 @@ void OnlineLocalizer::writeOutExpanded(const std::string &filename) const {
   std::fstream out(filename,
                    std::ios::out | std::ios::trunc | std::ios::binary);
   if (!patch.SerializeToOstream(&out)) {
-    printf("[ERROR][OnlineLocalizer] Couldn't open the file %s\n",
-           filename.c_str());
+    LOG(ERROR) << "Couldn't open the file" << filename;
     return;
   }
   out.close();
-  printf("[INFO][OnlineLocalizer] Wrote patch %s", filename.c_str());
+  LOG(INFO) << "Wrote patch " << filename;
 }
 
 // frontier picking up routine
 void OnlineLocalizer::matchImage(int quId) {
-  _expandedRecently.clear();
+  expandedRecently_.clear();
 
   std::unordered_set<Node> children;
-  if (_needReloc) {
-    _frontier = std::priority_queue<Node>(); // reseting priority_queue
-    printf("[INFO][OnlineLocalizer] RELOCALIZATION\n");
-    Node expandedNode = _currentBestHyp;
+  if (needReloc_) {
+    frontier_ = std::priority_queue<Node>(); // reseting priority_queue
```

Comment:
It's better to use the uniform initialization here to avoid the ["Most vexing parse"](https://en.wikipedia.org/wiki/Most_vexing_parse):
```c++
frontier_ = std::priority_queue<Node>{};
```